deploy/azureDeploy.js

```diff
@@ -14,13 +15,18 @@ class AzureDeploy {
 
     Object.assign(
       this,
+      createEventsBindings,
       loginToAzure,
       cleanUpFunctions,
       CreateResourceGroupAndFunctionApp,
       createFunctions
     );
 
     this.hooks = {
+      'before:package:initialize': () => BbPromise.bind(this)
```

Comment:
Should we be handling this logic as part of a new package command implementation? As opposed to handling package events in the deploy plugin? It probably doesn't matter too much, but it may be arguably cleaner to separate the package and deploy steps.